openapi.yml

```diff
@@ -603,12 +603,12 @@ components:
         cjsCode:
           type: string
           example: RR84070
-          description: Criminal Justice System offence code
-        courtDate:
-          type: string
+          description: Criminal Justice System offence code, only present for Offences retrieved from Prison API
+        courtDates:
+          type: array
           format: date
-          example: 2018-02-10
-          description: Latest court date associated with the offence
+          example: [2018-02-10, 2019-02-10]
+          description: Court dates associated with offences. Offences from Prison API will always be a list of 1 item. Offences from nDelius will have more than 1 court date.
```

Comment:
`type: array` requires you to specify an items sibling, so what you've put here won't render.
Consider something like:
```
        courtDates:
          type: array
          minItems: 1
          description: Court dates associated with offences. Offences from Prison API will always be a list of 1 item. Offences from nDelius will have more than 1 court date.
          items:
            type: string
            format: date
            example: [2018-02-10, 2019-02-10]
```